tests/test_bias.py

```diff
@@ -0,0 +1,182 @@
+import lsst.meas.algorithms as measAlg
+import lsst.utils.tests
+
+from lsst.pipe.tasks.repair import RepairTask
+from lsst.utils import getPackageDir
+
+
+class BiasTestCases(lsst.utils.tests.TestCase):
+
+    def setUp(self):
+        """Setup butler, and generate an ISR processed exposure.
+
+        Notes
+        -----
+        DMTN-101 4.1:
+
+        Process an independent bias frame through the ISR including
+        overscan correction and bias subtraction
+
+        """
+        repoDir = os.path.join(getPackageDir('ci_cpp_gen2'), "DATA")
+        calibDir = os.path.join(getPackageDir('ci_cpp_gen2'), "DATA", "calibs")
+        butler = dafPersist.Butler(repoDir, calibRoot=calibDir)
+
+        self.config = ipIsr.IsrTaskConfig()
+        self.config.doSaturation = True
+        self.config.doSuspect = True
+        self.config.doSetBadRegions = True
+        self.config.doOverscan = True
+        self.config.doBias = True
+        self.config.doVariance = True
+
+        self.config.doLinearize = False
+        self.config.doCrosstalk = False
+        self.config.doWidenSaturationTrails = False
+        self.config.doBrighterFatter = False
+        self.config.doDefect = False
+        self.config.doSaturationInterpolation = False
+        self.config.doDark = False
+        self.config.doStrayLight = False
+        self.config.doFlat = False
+        self.config.doApplyGains = False
+        self.config.doFringe = False
+        self.config.doMeasureBackground = False
+        self.config.doVignette = False
+        self.config.doAttachTransmissionCurve = False
+        self.config.doUseOpticsTransmission = False
+        self.config.doUseFilterTransmission = False
+        self.config.doUseSensorTransmission = False
+        self.config.doUseAtmosphereTransmission = False
+
+        self.isrTask = ipIsr.IsrTask(config=self.config)
+        # This is not an independent frame.
+        self.dataRef = butler.dataRef('raw', dataId={'detector': 0, 'expId': 2020012800007})
+        results = self.isrTask.runDataRef(self.dataRef)
+        self.exposure = results.outputExposure
+
+    def test_canary(self):
+        """Test for data value changes.
+        """
+        repoDir = os.path.join(getPackageDir('ci_cpp_gen2'), "DATA")
+        calibDir = os.path.join(getPackageDir('ci_cpp_gen2'), "DATA", "calibs")
+        butler = dafPersist.Butler(repoDir, calibRoot=calibDir)
+
+        dark = butler.get('bias', {'detector': 0, 'expId': 2020012800014})
+        m = hashlib.md5()
+        m.update(dark.getImage().getArray())
+
+        self.assertEquals(m.hexdigest(), '47b6328cbad5e4a212fc635e128d541e')
+
+    def test_independentFrameLevel(self):
+        """Test image mean.
+
+        Notes
+        -----
+        DMTN-101 4.2
+
+        Confirm that the mean of the result is 0 to within statistical
+        error
+        """
+        mean = afwMath.makeStatistics(self.exposure.getImage(), afwMath.MEAN).getValue()
+        print("4.2", mean)
```

Comment:
Remove this print statement. There's a couple of more below.

Reply:
Converted to msg arguments to the asserts (here and elsewhere), so a helpful message hopefully comes out if we fail a test.